Krypton/Notify.swift

```diff
@@ -115,7 +114,14 @@ class Notify {
                 content.subtitle = noteSubtitle
                 content.body = noteBody
                 content.sound = UNNotificationSound.default()
-                content.userInfo = ["session_display": session.pairing.displayName, "session_id": session.id, "request": request.object]
+                
+                let localRequest = LocalNotificationAuthority.VerifiedLocalRequest(alertText: noteBody,
+                                                                                   request: request,
+                                                                                   sessionID: session.id,
+                                                                                   sessionName: session.pairing.displayName)
+                
+                content.userInfo = (try? LocalNotificationAuthority.createSignedPayload(for: localRequest)) ?? [:]
```

Comment:
should an error here instead be displayed as an error in the notification content?

Reply:
👍